scripts/environment.d.ts

```diff
@@ -0,0 +1,48 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      CHAIN: string;
```

Comment:
these properties could be marked as optional, since there's no guaranty that they are defined. For ex: `CHAIN?: string;`